Debug App/Tests/Unit Tests/Primer/PrimerBancontactCardDataManagerTests.swift

```diff
@@ -11,7 +11,7 @@ import XCTest
 
 class PrimerBancontactCardDataManagerTests: XCTestCase {
     
-    private static let expectationTimeout = 1.0
+    private static let expectationTimeout = 5.0
```

Comment:
These were failing all through Friday so I bumped up the timeout, lets see how they do now. cc @jnewc

Reply:
Seems it was Xcode 15 that was flaky. Reverted to Xcode 14 for these (as it was before), and have a note to discuss in next chapter meeting